### DISCUSSION TITLE
What is timeline to stable Svg-Edit V7 version?

### DISCUSSION THREAD

**Discussion by dbInnermedia** [SVG-Edit/svgedit#490] [Q&A] [answered]
We are a small company that produces slides/reports for our customers. 
We are evaluating SVG editors to embed in our application.  SVG based slides are edited and used in presentations. We are currently evaluating SVG-Edit V6 and V7.  We are also looking at some other svg editors (commercial).  
Our application is a .Net WPf app, with an embedded web panel (using Microsoft's WebView2 this is Edge/Chromium browser window).  I've integrated V6 and V7 and have some issues with loading and reading documents to/from the embedded browser, but that is most likely my lack of knowledge.  The goal is to eventually shift our application to a web platform.  I'm running into editor issues in all the products I'm evaluating, so there is no clear winner, but we have a few months to evaluate and then integration would be fall time frame. So the question I have is what is the timeline for V7 stability?   Long term, If we did use SVG-Edit,  we would dedicate some resources to the editor.  My team's JS skill level is low currently. 
If anyone is attempting something similar, maybe we can trade notes and help each other. 
thanks,

**Comment by jfhenon** [comment, marked as answer]
Regarding your question on V7. It's getting there and I believe we can launch a beta in a few weeks. Regarding a "stable" version, it's honestly difficult to answer now. I hope it will be usable by June but we have changed so many things that it may take some time to fix bugs that will be discovered by the community. The next major item to develop before the beta release is the integration of a localization package (i18next or Localize) 

It would be a  pleasure to have support from your team on the project.
Kind regards
